sdk/src/wallet/store.rs

```diff
@@ -491,25 +490,13 @@ impl Store {
         Some(alias)
     }
 
-    /// Helper function to restore keypair given alias-keypair mapping and the
-    /// pkhs-alias mapping.
-    fn restore_keypair(
-        &mut self,
-        alias: Alias,
-        key: Option<StoredKeypair<common::SecretKey>>,
-        pkh: Option<PublicKeyHash>,
-    ) {
-        key.map(|x| self.keys.insert(alias.clone(), x));
-        pkh.map(|x| self.pkhs.insert(x, alias.clone()));
-    }
-
     /// Insert a new address with the given alias. If the alias is already used,
     /// will prompt for overwrite/reselection confirmation, which when declined,
     /// the address won't be added. Return the selected alias if the address has
     /// been added.
     pub fn insert_address<U: WalletIo>(
         &mut self,
-        alias: Alias,
+        mut alias: Alias,
```

Comment:
Now alias is both mutable and returned. That seems redundant to me.

Reply:
Only a local copy of the parameter is modified (so the caller doesn't see the mutation). You might be thinking of `alias: &mut Alias`...